docs/config.md

```diff
@@ -2,7 +2,8 @@
 
 These are the config settings available to jj/Jujutsu.
 
-The config settings are located at `~/.jjconfig.toml`. Less common ways to specify `jj` config settings are discussed in a later section.
+The config settings are located at `~/.jjconfig.toml`. Less common ways
+to specify `jj` config settings are discussed in a later section.
```

Comment:
It would have been nice is `.editorconfig` had a line-length config, but it doesn't seem like it. (Just a note; no changes requested.)

Reply:
I haven't even found a good way to get VS Code to format markdown. Vim/`fmt` work paragraph-by-paragraph.